exec/java-exec/src/main/java/org/apache/drill/exec/physical/rowSet/DirectRowSet.java

```diff
@@ -132,7 +131,7 @@ public RowSetReader reader() {
 
   @Override
   public SingleRowSet toIndirect() {
-    return new IndirectRowSet(this, Sets.<Integer>newHashSet());
+    return new IndirectRowSet(this, new HashSet<>());
```

Comment:
Should be immutable?

Reply:
Agree, replaced with `Collections.emptySet()`.